### ISSUE TITLE
Request for separating darkness event from core

### ISSUE THREAD

**Issue by freijie** [skyarkhangel/Hardcore-SK#627]
While being cool from coding point of view, Darkness event is really bugging me, there are several issues with it happening during early colony stages.

- Duration is so long that drafted pawns disable drafting automatically, making this event a micromanaging nightmare < biggest issue, colonists often decide to take a nap and die in a hallway with a couple of unlit tiles, and they run away from enemies to unlit areas sometimes as well.
- Audio effects sound out of place in rimworld, especially the sfx for event end.
- For me it bugged once by pausing the game after each tick, I coudlnt reproduce it to get the log, though. 

Basically it forces you to design the whole base with the main focus of countering this event's effects, which i find quite annoying considering how out of place this event is and how discruptive it is during early stages of colony expansion.

Would be nice to have it as a toggleable feature via hugslib menu or make it a separate mod. Together with super-animal events, actually - they are cool but still feel somewhat out of place in rimworld's sci-fi setting as well.

**Comment by k4iser88** [comment]
I disabled it in:
**C:\Program Files (x86)\Steam\steamapps\common\RimWorld\Mods\Core_SK\Defs\IncidentDefs**

in the "**Incidents_SK_events**" XML file you can actually set to "**0**" the chance of certain events, basically making them impossible to happen.

```
-<IncidentDef>
<defName>Darkness</defName>
<label>darkness</label>
<targetType>BaseMap</targetType>
<workerClass>SK.Events.IncidentWorker_BlackVoid</workerClass>

**<baseChance>0</baseChance>  <--- THIS IS THE PARAMETER YOU WANT TO CHANGE TO "0"**

<category>ThreatBig</category>
<minRefireDays>45</minRefireDays>
<earliestDay>60</earliestDay>
</IncidentDef>
```

**Comment by freijie** [comment]
Yep setting it to 0 or commenting out works, but updating from git is a pain this way,

**Comment by SmokeyUnicycle** [comment]
It breaks immersion and really just isn't very fun IMO. 

I just sigh when it occurs.

**Comment by Blothorn** [comment]
At the very least, it should have the description updated to specify "indoors in a lit area"; I recently got it for the first time, dutifully pulled all my pawns indoors, and suffered an immediate total colony wipe because I have been sloppy about indoor lighting.

Another thing that bugs me is that it is a death sentence for pawns far from base if they are slow or the map large. I can live with events like the razor rain that substantially threaten isolated pawns, because they are a bit random and you can at least minimize the risk by clever movement, opportunistic tunneling, etc. The darkness is a simple "The storyteller has decided that this pawn must die. So sorry."

**Comment by Rhyssia** [comment]
It shouldn't be too hard to separate Eternal Darkness and Darkness into the "Darkness_SK" mod

I agree that in most scenarios Darkness is a death sentence for a colony.

**Comment by SPomelov** [comment]
Agreed, it's an unbalanced trash not fitting the world. This is ok sometimes, but must be optional and even disabled by default.

**Comment by Schio248** [comment]
Apparently there is no IncidentDefs folder anymore. Can someone tell me how to disable the event now? I also think this should be disabled by default and optional.
